linode/manager
# fix: [M3-6720] - Restore DBaaS engine icons

## Description 📝
This fixes the issue where DBaaS engine icons are no longer being shown in the engine select drop-down.

I think our Select component used to expect a callback which returned an icon element for this purpose, but now expects the icon element to be passed directly, and this one drop-down may have slipped through the cracks.

## Preview 📷

| Before  | After   |
| ------- | ------- |
| <img width="491" alt="Screenshot 2023-06-23 at 12 00 18 PM" src="https://github.com/linode/manager/assets/97627410/c0f5521a-015b-446e-9074-47b9dfb6ef34"> | <img width="514" alt="Screenshot 2023-06-23 at 12 00 37 PM" src="https://github.com/linode/manager/assets/97627410/82a04141-0ad5-4fa6-aec1-0f685dc5a090"> |

## How to test 🧪
1. Navigate to DBaaS create page
2. Click the "Database Engine" select
3. Confirm that icons are shown in drop-down
